com.github.dictation_toolbox.Caster.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.github.dictation_toolbox.Caster
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: python3 -m dragonfly load _caster.py --engine kaldi --no-recobs-messages
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+
+build-options:
+    append-path: /usr/lib/sdk/rust-stable/bin
+
+cleanup:
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib/wx
+  - /lib64
+  - /share/aclocal
+  - /share/bakefile
+  - '*.la'
+  - '*.a'
+
+modules:
+
+  - shared-modules/glu/glu-9.json
+  - shared-modules/glew/glew.json
+
+  - name: xdotool
+    no-autogen: true
+    make-args:
+      - WITHOUT_RPATH_FIX=1
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz
+        sha256: 96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada
+  - name: portaudio
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/PortAudio/portaudio/archive/refs/tags/v19.7.0.tar.gz
+        sha256: 5af29ba58bbdbb7bbcefaaecc77ec8fc413f0db6f4c4e286c40c3e1b83174fa0
+
+  - name: wxWidgets
+    cleanup:
+      - /bin
+      - /include
+    config-opts:
+      - --with-opengl
+      - --disable-glcanvasegl
+    sources:
+      - type: git
+        commit: 085a136dcb11aca5b1102193f006f8056d5f0876
+        tag: v3.2.4
+        url: https://github.com/wxWidgets/wxWidgets
+        x-checker-data:
+          is-important: true
+          type: git
+          tag-pattern: ^v([\d\.]+)$
+
+  - python3-wxPython.yml
+
+  - caster.yml
+
+  - name: caster
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" -r requirements-mac-linux.txt
```

Comment:
Everything in here https://github.com/dictation-toolbox/Caster/raw/master/requirements-mac-linux.txt seems to be already in `caster.yml`? No point to have `-r`.


This is possibly also causing the issue you mentioned.

Reply:
So I just have to do: `pip3 install --no-index --find-links="file://${PWD}" caster`

?

Reply:
this is what I said on matrix. I guess it all got ignored.

Reply:
I haven't been ignoring you. I don't know if English is your first language, but you have been condescending and mildly rude to me since I joined the Matrix chat.

I have not said anything to avoid creating issues/drama, but I just wanted to bring this point up now so that I am not mistaken for being rude.

Reply:
> So I just have to do: `pip3 install --no-index --find-links="file://${PWD}" caster`
> 
> ?

Try `pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} .` or `python setup.py install --prefix=${FLATPAK_DEST}`